### DISCUSSION TITLE
Resources regarding Native Event Emitter

### DISCUSSION THREAD

**Discussion by shikharsrivastava23** [Agontuk/react-native-geolocation-service#319] [General]
Hey, @Agontuk thanks for this amazing library! This is a bit off topic, but I see that the library makes use of Native Event Emitters and involves managing of event listeners and subscriptions. 
I am trying to work on a navigation related project/module/library, and would like to learn more about NativeEventEmitter and how to handle event listeners when bridging native components. Any detailed guides and resources will be really helpful. 
Cheers.

**Comment by Agontuk** [comment]
Sorry for the late reply, you can check react native site for examples.
[https://reactnative.dev/docs/native-modules-android#sending-events-to-javascript](https://reactnative.dev/docs/native-modules-android#sending-events-to-javascript)
[https://reactnative.dev/docs/native-modules-ios#sending-events-to-javascript](https://reactnative.dev/docs/native-modules-ios#sending-events-to-javascript)
